duckduckgo/iOS
# Autofill settings list deletion undo

Task/Issue URL: https://app.asana.com/0/0/1202808746542215/f
Tech Design URL:
CC:

**Description**:
Originally I was going to bundle all the settings list changes into one PR, but I think this is potentially worth some separate discussion. 

This PR adds a toast when ever you delete a login item from the settings list, with an undo feature that allows you to undelete them.

I'll comment in line, but even though I wrote it, I'm not sure I love where this lives and think it's at least worth a conversation.




**Steps to test this PR**:
1. Ensure existing toasts haven't been broken. Things this PR could have accidentally affect are both the positioning relative to the bottom of the screen, and the actions when the button on them is pressed.
2. Delete some logins, check you get the toast and it's not too high up the screen. Try undoing some of them, they should immediately reappear in the list. Try editing those and check they work correctly



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
